### ISSUE TITLE
theme preview in user settings

### ISSUE THREAD

**Issue by uhoreg** [vector-im/riot-web#3795]
It's probably not so important right now, when there are only two themes whose names basically tell the user the difference between them, but once there are more themes, it would be good to have a preview of each theme showing various UI elements (text, background, button, etc.) so that users can see what they all look like at a glance to make it easier for them to pick.

**Comment by lampholder** [comment]
Agreed on both the desirability of this feature and its hugely increased value when we have more themes :)
